src/mono/mono/btls/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.8.10)
+cmake_minimum_required (VERSION 3.14.5)
```

Comment:
please revert the change here, this folder isn't used by netcore mono (we should probably delete it from the mirror)

Reply:
Sure, I have reverted `btls` directory to `HEAD~1`.